scala/scala_maven_import_external.bzl

```diff
@@ -0,0 +1,295 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rules for defining external Java dependencies.
+
+java_import_external() replaces `maven_jar` and `http_jar`. It is the
```

Comment:
is this just a temporary copy paste? can we not get this with a remote dependency?

Reply:
I can make the existing java_import_external serve your use case, if you'd prefer.

Reply:
Yes this is a temporary copy paste.
We're starting out this way to validate the differences are indeed small instead of iterating between this repository and bazel itself.
The general plan is to generalize with as few changes as possible and then port 80%-90% of the code back to bazel